modules/ROOT/pages/prerequisites/prerequisites.adoc

```diff
@@ -319,15 +321,32 @@ NFS Notes::
 +
 [NOTE]
 ====
-When using NFS, you have to take care that the NFS server provides `Extended Attributes`.
+When using NFS, you have to take care that the NFS server *AND* the NFS client provides `Extended Attributes`.
 
-NFS storage based on Linux servers::
-Extended attributes are supported by NFS starting with kernel version 5.9, therefore the NFS server has to run on a system with kernel 5.9 or higher. To check, run the command `uname -a` on the NFS server and compare the displayed version number.
+NFS Storage Based on Linux Servers::
+When using a kernel version 5.9 or higher, extended attributes for NFS server and NFS client are part of the system. To check, run the command:
++
+[source,bash]
+----
+uname -r
+----
++
+On the system providing the NFS server AND on the NFS client check the displayed version number.
+
+NFS Client::
+If you have an NFS server capable of extended attributes but you are unsure if your client accessing the server supports it, check the *nfs-utils* or *nfs-common* package version of your NFS client with the command:
++
+[source,bash]
+----
+mount.nfs -V
+----
++
```

Comment:
Maybe an bash example how to check if extended attributes would be helpful, sth like:

```
$> touch foo.txt
$> attr -s foo -V bar foo.txt
Attribute "foo" set to a 3 byte value for foo.txt:
bar
$> attr -g foo foo.txt
Attribute "foo" had a 3 byte value for foo.txt:
bar
```

Reply:
added, pls check